freetextresponse/private/view.less

```diff
@@ -0,0 +1,77 @@
+.freetextresponse_block {
```

Comment:
I also think the `_block` suffix is cargo cult and could probably be dropped.

Thoughts?